src/app/components/wrapper-components/tutorial-game-wrapper/tutorial-game-wrapper.wrapper.component.spec.ts

```diff
@@ -819,7 +822,7 @@ describe('TutorialGameWrapperComponent (wrapper)', () => {
                 componentTestUtils.findElement('#currentMessage').nativeElement.innerHTML;
             expect(currentMessage).toBe(expectedMessage);
         }));
-        it('When unwanted click, and no move done, restart should not be needed', fakeAsync(async() => {
+        it('When unwanted click, and no move done, show restart with possibility of seeing solution', fakeAsync(async() => {
```

Comment:
show restart button*
Sinon c'est semi-confusant non ?

Reply:
'button' ajouté. Sinon tu veux dire confusant de montrer le bouton « restart » ? L'idée c'est que si tu piges pas où cliquer, tu peux voir la solution donc

Reply:
Show restart button (with possibility of seeing solution) when unwanted click with no move done
Should show failure message after step failure (one of several clics)
Should show success message after step success (one of several clics)
should show solution when asking for it

Quand tu rate ton clic, tu sais recliquer ?
J'ai l'impression qu'après avoir raté son clic faudrait reclicker sur "redémarrer" ou "voir soluce" d'office, non ?

Reply:
hm c'est déjà les titres que j'avais (à deux mots près) ou bien j'ai raté un truc ?

Et pour le moment, si tu rate ton clic tu rate l'étape, donc tu peux plus cliquer (« l'étape est finie »), ça requiert un refactor plus gros pour pouvoir réessayer sans cliquer.

Reply:
haha c'est les titres au mot près, je les collais-compilait pour faire le bilan de nos specs (dans un .spec.ts, tellement bien nommé, tellement BDD)

Par tu peux plus clicker ça veut dire quoi ? ctrl+F5 et tu recommence le tuto :D ? Ou bien y a quand même les boutons "ré-essayer/redémarrer" ?

Reply:
(up)